cmd/ddev/cmd/testdata/TestRealWorldDependencies/mock_redis_commander/docker-compose.redis-commander.yaml

```
@@ -0,0 +1,10 @@
+#ddev-generated
+# Mock Redis Commander Docker Compose configuration
+services:
+  redis-commander:
+    image: rediscommander/redis-commander
+    container_name: ddev-${DDEV_SITENAME}-redis-commander
+    ports:
+      - "8081"
+    environment:
+      - REDIS_HOSTS=redis:redis:6379
```

Just to make it look real:
```suggestion
      - REDIS_HOST=redis
```